src/lib/apis/auth.api.ts

```diff
@@ -17,12 +17,16 @@ export const createChallenge = async (publicKey: string) => {
     }).then(res => res.json())
 };
 
-export const verifyNewUser = async (username: string, password: string): Promise<ChallengeResponse> => {
-    return await fetch(`${baseUrl}/user/register`, {
+export const getChallenge = async (username: string, tempPassword: string): Promise<ChallengeResponse> => {
```

Comment:
should we call it something like getRegsitrationChallenge ??

Reply:
will do